about_me.css

```diff
@@ -0,0 +1,92 @@
+* {
+    box-sizing: border-box;
+}
+
+body {
+    margin: 0;
+}
+
+@media screen and (max-width: 500px) {
+    body {
+        background-color: #8b0000;
+    }
+}
+
+nav {
+    background-color: #000000;
+    height: 80px;
+    display: block;
+    padding: 0 15px;
+}
+
+.container {
+    float: right;
+    margin-top: 26px;
+
+}
+
+.container a {
+    color: black;
+    text-decoration: none;
+    text-align: center;
+    background-color: #8b0000;
+    border-radius: 15px;
+    box-shadow: 0 9px #999;
```

Comment:
to add blurring: add a third pixel quantity to add blurred shadow effect. higher is more blurred.